core/src/main/java/org/apache/camel/kafkaconnector/CamelSinkConnectorConfig.java

```diff
@@ -38,10 +38,16 @@
     public static final String CAMEL_SINK_URL_DOC = "The camel url to configure the destination. If this is set " + CAMEL_SINK_COMPONENT_CONF
             + " and all the properties starting with " + CamelSinkTask.getCamelSinkEndpointConfigPrefix() + ".<" + CAMEL_SINK_COMPONENT_CONF + " value> are ignored.";
 
+    // do not log record's payload by default, as it may contain sensitive information
+    public static final Boolean CAMEL_SINK_LOG_PAYLOAD_DEFAULT = false;
+    public static final String CAMEL_SINK_LOG_PAYLOAD_CONF = "camel.source.logPayload";
```

Comment:
This I think should be `"camel.sink.logPayload"` ?

Reply:
Thanks